draft-packages/title-block-zen/KaizenDraft/TitleBlockZen/TitleBlockMenuItem.tsx

```diff
@@ -14,11 +14,12 @@ export type TitleBlockMenuItemProps =
 export const TitleBlockMenuItem = (
   props: TitleBlockMenuItemProps
 ): JSX.Element | null => {
-  const className = classnames(styles.menuItem, {
-    [styles["menuItem--destructive"]]: styles.destructive,
-    [styles["menuItem--disabled"]]: styles.disabled,
-    [styles["menuItem--active"]]: styles.isActive,
-  })
+  const className = classnames(
+    styles.menuItem,
+    styles.destructive && styles["menuItem--destructive"],
+    styles.disabled && styles["menuItem--disabled"],
+    styles.isActive && styles["menuItem--active"]
```

Comment:
Wait what? What is happening here? 😅

Reply:
@HeartSquared hurt herself in her confusion